### PR TITLE
Made DetailView scrollable!

### PR DESCRIPTION
**Pull request by mathieupost**
You can see every genome and label in a node now! Amazing!

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/detailview/DetailView.java [hunk 1]
```diff
@@ -19,14 +22,26 @@
  */
 public class DetailView extends JPanel {
 	private static final int BORDER = 10;
-	private static final int MAX_ROWS = 5;
 	private static final int POSITION_OFFSET = 30;
+	public static final String REF_GENOME = "TKK_REF";
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/detailview/DetailView.java]: is the substring REF not sufficient as reference genome? Or is TKK an essential part?

**Comment by Delfil** [comment]
The detail view labels contains duplicates.

**Comment by kwendel** [comment]
looks cool!
maybe make the view box is little bit bigger with some padding on the sides? this increases readability

**Comment by kwendel** [comment]
NicE!

**Pull request merged**